doc/ha/dev-setup.md

```diff
@@ -70,32 +70,54 @@ ziti pki create server --pki-root ./pki --ca-name ctrl3 --dns localhost --ip 127
 ## Running the Controllers
 
 1. The controller configuration files have relative paths, so make sure you're running things from this directory.
-2. Start all three controllers
-    1. `ziti controller run ctrl1.yml`
-    2. `ziti controller run ctrl2.yml`
-    3. `ziti controller run ctrl3.yml`
-    4. All three are configured with `minClusterSize` of 3, so they will wait to be joined to a raft cluster
-    5. The ctrl1.yml config file has the other two controllers as bootstrap members, so when it starts the first controller will start trying form the raft cluster.
-3. Initialize the edge using the agent
-    1. `ziti agent controller init <pid of any controller> admin admin 'Default Admin'`
-    2. You can of course use different values if you desire
-
-You should now have a three node cluster running. You can log into each controller individually.
+2. Start all three controllers.
+
+    ```bash
+    ziti controller run ctrl1.yml
+    ziti controller run ctrl2.yml
+    ziti controller run ctrl3.yml
+    ```
+
+    All three are configured with `minClusterSize` of 3, so they will wait to be joined to a raft cluster.
+    The ctrl1.yml config file has the other two controllers as bootstrap members, so when it starts the first
+    controller will start trying form the raft cluster.
+
```

Comment:
Once this: https://github.com/openziti/ziti/pull/1524 is merged, the `cluster add` commands should no longer be necessary, so we can probably either take it out or make a note that you could use this method instead of the bootstrapMembers config option.

Reply:
I moved those down to the bottom as a troubleshooting note.